### PR TITLE
clarify surprising return value

### PR DESCRIPTION
**Pull request by Boldewyn**
### Description

`document.fonts.check()` returns `true` for non-existing or nonsensical font names. Cf. https://bugzilla.mozilla.org/show_bug.cgi?id=1252821#c3 and https://drafts.csswg.org/css-font-loading/Overview.bs for the details.

### Motivation

The continued return value of `true` is surprising for people being confronted with `document.fonts.check()`, if they want to use it to test for anything else than their own fonts.

### CODE REVIEW

File: files/en-us/web/api/fontfaceset/check/index.md [hunk 1]
```diff
@@ -35,6 +35,14 @@ check(font, text)
 
 A {{jsxref("Boolean")}} value that is `true` if the font list is available.
 
+The method returns `true` also, if the font list contains system or non-existing fonts. That is, the call
+
+```js
+console.log(document.fonts.check("12px NonExistingFont"));
+```
+
+will print `true`. This somewhat counterintuitive behavior prevents user tracking by locally installed fonts, though.
```
Comment (Josh-Cena) [inline files/en-us/web/api/fontfaceset/check/index.md:44]:
```suggestion
The method returns `true` if the font list contains system or non-existing fonts. This prevents user tracking by locally installed fonts.
```

The "return value" needs to be succinct. If you wish, you may add an example below.

Comment (Boldewyn) [reply files/en-us/web/api/fontfaceset/check/index.md]: Thank you! I’ve updated the patch with your text and added an additional example.

**Comment by Josh-Cena** [comment]
Interestingly, this doesn't match the behavior in my Chrome, although it does in Firefox:

image

Is there some user preference?

**Comment by Boldewyn** [comment]
Apparently this is a bug in Chrome, as the spec author noted and a later commentator annotated: https://bugs.chromium.org/p/chromium/issues/detail?id=591602#c12
What is the procedure to continue in this case? Unfortunately the spec is still a working draft, so a moving target. Current state: https://github.com/w3c/csswg-drafts/blob/main/css-font-loading-3/Overview.bs#L921-L923

**Comment by Josh-Cena** [comment]
I would leave this to the other reviewers. I think a BCD entry at most would suffice. The content, editorial wise, LGTM

**Review by wbamberg — CHANGES_REQUESTED** [review]
I hope you don't mind the drive-by copy-edits.

> Apparently this is a bug in Chrome, as the spec author noted and a later commentator annotated: https://bugs.chromium.org/p/chromium/issues/detail?id=591602#c12 What is the procedure to continue in this case? Unfortunately the spec is still a working draft, so a moving target. Current state: https://github.com/w3c/csswg-drafts/blob/main/css-font-loading-3/Overview.bs#L921-L923

I think this should be a note in the browser compat, under "chrome": https://github.com/mdn/browser-compat-data/blob/370b0e34d822aa52995468480bce626137bc58d0/api/FontFaceSet.json#L137 .

File: files/en-us/web/api/fontfaceset/check/index.md [hunk 2]
```diff
@@ -45,6 +47,12 @@ console.log(document.fonts.check("12px courier"));
 console.log(document.fonts.check("12px MyFont", "ß"));
 ```
 
+The use of a non-existing font in the font specification results in a return value of `true`. The following example line will print `true`.
+
+```js
+console.log(document.fonts.check("12px NonExistingFont"));
+```
```
[the remark this answers is not in the document]
Comment (Boldewyn) [reply files/en-us/web/api/fontfaceset/check/index.md]: Good two suggestions! Thank you!

**Review by wbamberg — APPROVED** [review]
Thank you @Boldewyn , and thanks also for opening the BCD PR.

I'm happy with this but leaving it open for @Josh-Cena 's review :).

**Review by Josh-Cena — APPROVED** [review]
LGTM (I only make random drive-by reviews for non-JS stuff)

**Pull request merged**